pom.xml

```diff
@@ -100,6 +100,8 @@
     <skipUTs>${skipTests}</skipUTs>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <main.basedir>${project.basedir}</main.basedir>
+    <mr.bundle.avro.scope>provided</mr.bundle.avro.scope>
```

Comment:
do these need to be declared at the root pom level or can it be localized to the bundle pom?

Reply:
Unfortunately yes. If we move it to bundle POM, then `aws-emr-profile` does not work, and only `mr-bundle-shade-avro` works.

Maven possibly looks for the property to be present in the same pom file or atleast be inherited from its parent in order to override it using Profiles.